package.json

```diff
@@ -35,7 +35,7 @@
     "express-jwt": "5.3.1",
     "inflected": "2.0.4",
     "ip-utils": "git+https://github.com/ForestAdmin/ip-utils.git#5f88562ba53fedcdc0374937fca0fdb71fa4923c",
-    "jsonapi-serializer": "3.4.1",
+    "jsonapi-serializer": "3.6.1",
```

Comment:
This fix some issue with the serializer and allow to return null from relationshipLinks function and not include the link in this case